test/inductor/test_cpu_repro.py

```diff
@@ -3422,7 +3422,6 @@ def fn():
                 dtype if dtype else torch.float32,
             )
 
-    @config.patch("cpp.enable_tiling_heuristics", False)
     def test_group_norm_vec(self):
```

Comment:
I think the UT also need to check the vectorization is as expected instead of scalarization as fallback

Reply:
we have checked the number of vec kernels here: https://github.com/pytorch/pytorch/pull/135574/files/650609529833142b18d72b7f35ae199228fb896c#diff-042e454a3b5e4ae5aea2ff41066b5646ab068ca92b7e0adb5f9a81c6d7cec422L3446
I removed the patch that disabled the tiling select heuristic, now the UT can correctly check the vectorization is as expected.